src/main/webapp/default-site/scripts/api/SiteServices.groovy

```diff
@@ -69,11 +69,11 @@ class SiteServices {
 
     static createSiteWithRemoteOption(context, siteId, sandboxBranch, description, blueprint, remoteName, remoteUrl,
                                       remoteBranch, singleBranch, authenticationType, remoteUsername, remotePassword,
-                                      remoteToken, remotePrivateKey, createOption, params, createAsOrphan) {
+                                      remoteToken, remotePrivateKey, createOption, params, createAsOrphan, name) {
```

Comment:
Move siteName to the correct/expected order in the parameter list. (change `name` to siteName)

Reply:
should the parameter in the request also be `siteName` instead of `name`?

Reply:
No need to change the external API in V1. I'm just looking to change the internal names to be consistent.